docs/reference/api/usb/USBAudio.md

```diff
@@ -2,22 +2,59 @@
 
 <span class="images">![](https://os.mbed.com/docs/v5.9/feature-hal-spec-usb-device-doxy/class_u_s_b_audio.png)<span>USBAudio class hierarchy</span></span>
 
-You can use the USBAudio interface to send and receive audio across USB. By selecting a USBAudio device as your PC's speaker output, you can send audio data. You can also use it as your PC's microphone and receive sound sent by USBAudio.
+You can use the USBAudio interface to send and receive audio data over USB. Once a USB program is loaded onto the Mbed board, you can send audio data to your PC by selecting **Mbed Audio** as your PC's microphone input, and you can have the Mbed board receive audio data from your PC by selecting **Mbed Audio** as your PC's speaker output.
 
 ### USBAudio class reference
 
 [![View code](https://www.mbed.com/embed/?type=library)](https://os.mbed.com/docs/v5.9/feature-hal-spec-usb-device-doxy/class_u_s_b_audio.html)
 
-### USBAudio example
+### USBAudio square wave example
```

Comment:
Code comments are good about what the example does. It would be great to have some introduction to this (or replicate the same info) in the markdown as well..to set the scene to the reader..